supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -379,6 +379,28 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *See also*:
 
+[discrete]
+[[operator-hub]]
+==== image:images/yes.png[yes] OperatorHub (noun)
+*Description*: In red Hat OpenShift, the "OperatorHub" is a central location where you can find a wide array of useful Operators built by the community.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[operator-lifecycle-manager]]
+==== image:images/yes.png[yes] Operator Lifecycle Manager (OLM) (noun)
```

Comment:
Can you remove "(OLM)" here? We want to try to reserve the parentheses for the part of speech. The "(OLM)" part is referenced in the sentence below, so that should be sufficient.